src/Resolver/RememberDeviceResolver.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/*
+ * This file is part of the Connect Holland Secure JWT package and distributed under the terms of the MIT License.
+ * Copyright (c) 2020 Connect Holland.
+ */
+
+namespace ConnectHolland\SecureJWTBundle\Resolver;
+
+class RememberDeviceResolver
+{
+    private $configuration;
+
+    private $debug;
```

Comment:
what is the `$debug` property used for?

Reply:
Me and Bas copied it from the example we used: https://github.com/ConnectHolland/user-bundle

However, we didn't seem to use/need it so removed it in my latest commit